.prettierignore

```diff
@@ -0,0 +1,4 @@
+dist
+dist6
+api-docs
+*.json
```

Comment:
This line is for IDE extensions - it's telling them to don't apply prettier formatting when saving JSON files. It works for me in VS Code.